docs/jwst/tweakreg/README.rst

```diff
@@ -191,7 +191,19 @@ Parameters used for absolute astrometry using a reference catalog:
 * ``sigma``: A positive `float` indicating the clipping limit, in sigma units,
   used when performing fit. (Default=3.0)
 
-Parameters used for absolute astrometry using a reference catalog:
+**Absolute Astrometric fitting parameters:**
+
+Parameters used for absolute astrometry to a reference catalog.
+
+* ``abs_refcat``: String indicating what astrometric catalog should be used.
+  Currently supported options: `None`, '', 'GAIADR2' and 'GAIADR1' or a path
```

Comment:
Minor nit pick, but it seems to me that a more logical order for the options would be something like GAIADR1, GAIADR2, path to a user-supplied cat, None, or '' (i.e. leave the null options till the end).